includes/functions-llms-blocks.php

```diff
@@ -66,7 +67,12 @@ function llms_blocks_is_post_migrated( $post ) {
 		if ( llms_blocks_is_classic_enabled_for_post( $post_id ) ) {
 			$ret = false;
 		} else {
-			$ret = llms_parse_bool( get_post_meta( $post_id, '_llms_blocks_migrated', true ) );
+			// Checking if the post is an auto-draft (new post without any postmeta) and the Classic Editor is not active.
+			if ( ! wp_is_post_revision( $post_id ) && 'auto-draft' === get_post_status( $post_id ) && ! class_exists( 'Classic_Editor' ) ) {
```

Comment:
cat a revision be an auto-draft?
also you can have the classic editor enabled but still use the block editor for a course/lesson, so this check it's not enough.
Additionally we're changing here the logic of a function `llms_blocks_is_post_migrated()` which is used in several places, for the purpose of fixing a specific issue, are we sure it doesn't impact on anything else?
I would rather scope the change for that case only. What do you think?

Reply:
@eri-trabiccolo 

No, revision cannot be an auto-draft, as far as I know. That's a bit wrong condition.
I've opened a [new PR](https://github.com/gocodebox/lifterlms/pull/2412) in the LifterLMS, if that's correct, we can close this one :D